.github/workflows/dp-workflow.yml

```diff
@@ -179,7 +178,6 @@ jobs:
         with:
           flags: unittests,active-mode-controller
           name: codecov-active-mode-controller
-          fail_ci_if_error: false
```

Comment:
There are two more cases where `fail_ci_if_error: false` is set in this file, could we also remove these?

Reply:
You are correct. Also removed. Thank you. :slightly_smiling_face: